docs/mine/lotus/split-markets-miners.md

```diff
@@ -10,7 +10,9 @@ breadcrumb: 'Splitting main miner and markets service processes'
 
 ## Background
 
-Lotus v1.11.0 introduced the notion of `subsystems` in the `lotus-miner` process. Currently there are 4 subsystems, that can be enabled or disabled in the `config.toml` file:
+> Note: this feature is to be introduced in Lotus v1.11.1
```

Comment:
```suggestion
> Note: this feature is available starting with Lotus v1.11.1
```

Rationale: the future tense makes sense now, but we won't come and update this when it becomes the past.